source/standard_scores.Rmd

```diff
@@ -0,0 +1,1523 @@
+```
+
+```{r}
+# Rank of McCarthy's district in terms of median household income.
+kom_rank = 295
+# Index (position) of McCarthy's value in the "incomes" vector.
+# This is the same as the rank.
+kom_index = kom_rank
+```
+
+Now we have the index (position) of KOM's value, we can find the household
+income for his district from the `incomes` {{< var array >}}:
+
+```{python}
+# Show the median household income from McCarthy's district
+# by indexing into the "incomes" array:
+kom_income = incomes[kom_index]
+kom_income
+```
+
+```{r}
+# Show the median household income from McCarthy's district
+# by indexing into the "incomes" vector:
+kom_income <- incomes[kom_index]
+kom_income
+```
+
+Here is the corresponding index and `incomes` value for AOC:
+
+```{python}
+# Index (position) of AOC's value in the "incomes" array.
+aoc_rank = 81
+aoc_index = aoc_rank - 1
+# Show the median household income from AOC's district
+# by indexing into the "incomes" array:
+aoc_income = incomes[aoc_index]
+aoc_income
+```
+
+```{r}
+# Index (position) of AOC's value in the "incomes" array.
+aoc_rank = 81
+aoc_index = aoc_rank
+# Show the median household income from AOC's district
+# by indexing into the "incomes" array:
+aoc_income <- incomes[aoc_index]
+aoc_income
+```
+
+Notice that we fetch the same value for median household income from `incomes`
+as you see in the corresponding rows.
+
+## Comparing values with ranks and quantile positions
+
+We have KOM's and AOC's district median household income values, but our next
+question might be — how unusual are these values?
+
+Of course, it depends what we mean by *unusual*.   We might mean, are they
+greater or smaller than most of the other values?
+
+One way of answering that question is simply looking at the rank of the values.
+If the rank is lower than $\frac{441}{2} = 220.5$ then this is a district with
+higher median income than most districts.  If it is greater than $220.5$ then
+it has lower median income than most districts.  We see that KOM's district,
+with rank `r get_var('kom_rank')` is wealthier than most, whereas AOC's
+district (rank `r get_var('aoc_rank')`) is poorer than most.
+
+But we can't interpret the ranks without remembering that there are 441 values,
+so — for example - a rank of 81 represents a relatively low value, whereas one
+of 295 is relatively high.
+
+We would like some scale that tells us immediately whether this is a relatively
+low or a relatively high value, without having to remembering how many values
+there are.
+
+This is a good use for *quantile positions* (QPs).  The QP of a value tells you
+where the value ranks relative to the other values, on a scale from $0$ through
+$1$. A QP of $0$ tells you this is the lowest-ranking value, and a QP of $1$
+tells you this is the highest-ranking value.
+
+We can calculate the QP for each rank.  Think of the low-to-high ranks as being
+a line starting at 1 (the lowest rank — for the lowest median income) and going
```

Comment:
Perhaps flip those wors around: the lowest median income, with rank 0

Reply:
The confusing bit for me was that "1" may be the lowest rank in R, but not in Python, where it would be zero (I think?). So I first was thinking of entirely the wrong thing.

Reply:
Ah - actually - 1 is also the lowest rank in Python, at least using `sps.rankdata`.     Can you think of a better way of phrasing it?